POUF1.md

```diff
@@ -0,0 +1,442 @@
+* POUF: 1
+* Title: Uptane Reference Implementation POUF
+* Version: 1
+* Last-Modified: 17-07-2019
+* Author: Marina Moore
+* Status: Draft
+* Uptane Version Implemented: 1.0.0
+* Created: 14-03-2019
+
+# Abstract
+This document describes the Protocols, Operations, Usage, and Formats (POUF) for the Uptane reference implementation written in Python by NYU. POUFs are designed to allow interoperable Uptane implementations. Any implementer who wishes to interoperate with the reference implementation should implement the features and data structures described in this document.
+
+**Note that this implementation is not meant to be used in practice**. It is a guide for implementers to understand how the system should function.
+
+The reference implementation includes all required and most optional features as a reference for others looking to implement Uptane.
+
+# Protocols
+This POUF uses ASN.1/DER encoding for transmitting files. The details of the file formats are described in {Formats} in ASN.1 notation. Note that improper decoding of ASN.1 may lead to arbitrary code execution or Denial-of-Service attacks. Any implementers of this POUF should be aware of these attacks and use a decoder which has been subject to appropriate security analysis.
+
+The Primary ECU communicates with the Director repository, Image repository, and time server using http post requests over [XML-RPC](http://xmlrpc.scripting.com/spec.html). In this communication, the Primary ECU acts as an XML-RPC client and the Director repository, Image repository, and time server act as XML-RPC servers. The requests supported by these servers are described in the Message Handler Table.
+
+This POUF does not specify communication between Primary and Secondary ECUs. This communication is left up to the implementer, but should follow the requirements of the standard. For more information about internal vehicle communication, see the [Deployment Considerations](https://uptane.github.io/deployment/introduction.html).
```

Comment:
If this is true then how can this be interoperable?  Does it qualify as a POUF?  Why is it okay to miss this?